docs/operator-manual/server-commands/argocd-repo-server.md

```diff
@@ -0,0 +1,32 @@
+## argocd-repo-server
+
+Run argocd-repo-server
+
+### Synopsis
+
+Run argocd-repo-server
```

Comment:
Same comment as 'argocd-server' synopsis, maybe something like:

"This command starts the repository server, which is an internal service which maintains a local cache of the Git repository holding the application manifests, and is responsible for generating and returning the Kubernetes manifests. This server may be customized via the command parameters below."

Reply:
Updated